test/index.js

```diff
@@ -26,739 +26,615 @@ describe('Token', function () {
 
-      server.route([
+    before(async function () {
+      await newServer.register(require('../'))
+      newServer.auth.strategy('default', 'jwt', { key: privateKey, validateFunc: loadUser, issuer: [issuer,'issuer2','issuer3']});
+      newServer.auth.default('default');
+
+      newServer.route([
         { method: 'POST', path: '/token', handler: tokenHandler, config: { auth: 'default' } }
       ]);
+
+      await newServer.start();
     });
-    
-    it('fails if token issuer is empty', function (done) {
-  
+
+    it('fails if token issuer is empty', async function () {
       var request = { method: 'POST', url: '/token', headers: { authorization: tokenHeader('john') } };
 
-      server.inject(request, function (res) {
-        expect(res.result.message).to.equal(jwtErrorPrefix + 'jwt issuer invalid. expected: ' + issuer + ',issuer2,issuer3');
-        expect(res.statusCode).to.equal(401);
-        done();
-      });
+      const res = await newServer.inject(request);
+      expect(res.result.message).to.equal(jwtErrorPrefix + 'jwt issuer invalid. expected: ' + issuer + ',issuer2,issuer3');
+      expect(res.statusCode).to.equal(401);
     });
 
-    it('fails if token issuer is invalid', function (done) {
-  
+    it('fails if token issuer is invalid', async function () {
       var request = { method: 'POST', url: '/token', headers: { authorization: tokenHeader('john', {issuer:'https://invalid.issuer'}) } };
 
-      server.inject(request, function (res) {
-        expect(res.result.message).to.equal(jwtErrorPrefix + 'jwt issuer invalid. expected: ' + issuer + ',issuer2,issuer3');
-        expect(res.statusCode).to.equal(401);
-        done();
+      const res = await newServer.inject(request);
+      expect(res.result.message).to.equal(jwtErrorPrefix + 'jwt issuer invalid. expected: ' + issuer + ',issuer2,issuer3');
+      expect(res.statusCode).to.equal(401);
       });
-    });
 
-    it('works if token issuer contains one of the expected issuers valid', function (done) {
-  
+    it('works if token issuer contains one of the expected issuers valid', async function () {
+
       var request = { method: 'POST', url: '/token', headers: { authorization: tokenHeader('john', {issuer: issuer}) } };
 
-      server.inject(request, function (res) {
-        expect(res.result).to.exist;
-        expect(res.statusCode).to.equal(200);
-        done();
-      });
+      const res = await newServer.inject(request);
+      expect(res.result).to.exist;
+      expect(res.statusCode).to.equal(200);
     });
 
   });
 
   describe('when RS256 is specified as algorithm for validation', function(){
-    var issuer = 'http://expected.issuer'; 
-    
-    var server = new Hapi.Server({ debug: false});
-    server.log(['error', 'database', 'read']);
-    server.connection();
-    server.register(require('../'), function (err) {
-      expect(err).to.not.exist;
+    var newServer = new Hapi.Server({ debug: false });
```

Comment:
We removed the `.log` call here, but kept it on the prev one.

Reply:
Removed